pkg/plugin/admissioncontrollers/webhook/webhook.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2019 Cloudera, Inc.  All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"github.com/cloudera/yunikorn-core/pkg/log"
+	"github.com/golang/glog"
+	"go.uber.org/zap"
+	"net/http"
+	"os"
+	"os/signal"
+	"path/filepath"
+	"syscall"
+)
+
+const (
+	HttpPort    = 8443
+	tlsDir      = `/run/secrets/tls`
+	tlsCertFile = `cert.pem`
+	tlsKeyFile  = `key.pem`
+)
+
+func main() {
+	certPath := filepath.Join(tlsDir, tlsCertFile)
+	keyPath := filepath.Join(tlsDir, tlsKeyFile)
+	pair, err := tls.LoadX509KeyPair(certPath, keyPath)
+	if err != nil {
+		glog.Fatal("Failed to load key pair: %v", err)
```

Comment:
We should not be using glog, replace with zap

Reply:
Fixed